server/src/main/java/com/linecorp/centraldogma/server/internal/api/ProjectServiceV1.java

```diff
@@ -145,17 +144,10 @@ public CompletableFuture<ProjectDto> createProject(CreateProjectRequest request,
      * GET /projects/{projectName}
      *
      * <p>Gets the {@link ProjectMetadata} of the specified {@code projectName}.
-     * If a {@code checkPermissionOnly} parameter is {@code true}, it is returned whether the user has
-     * permission to read the metadata of the specified {@code projectName}.
      */
     @Get("/projects/{projectName}")
-    @RequiresRole(roles = { ProjectRole.OWNER, ProjectRole.MEMBER })
-    public CompletableFuture<ProjectMetadata> getProjectMetadata(
-            @Param String projectName,
-            @Param("checkPermissionOnly") @Default("false") boolean isCheckPermissionOnly) {
```

Comment:
This parameter was introduced [here](https://github.com/line/centraldogma/pull/133/files#diff-da7ebe272e478dcd441cb91f2cd7680672cfabd20b271a9c970b7bb14facc52bR7) but has not been used anywhere. Let me just remove that.